README.md

```diff
@@ -118,38 +118,6 @@ calling the `emogrify` method:
 * `$emogrifier->addExcludedSelector(string $selector)` - Keeps elements from
   being affected by emogrification.
 
-## Installing with Composer
```

Comment:
This heading also needs reming from the Table of Contents.

Reply:
Done.